app/views/hyrax/base/_form_share.html.erb

```diff
@@ -1,7 +1,7 @@
-<%# OVERRIDE: Hyrax 2.5.1 to use Hyrax::Groups for groups select box %>
+<%# OVERRIDE: Hyrax v5.0.0rc2 to use Hyrax::Groups for groups select box %>
 
 <p><%= t('.directions') %></p>
-<h2><%= t('.add_sharing') %></h2>
+<h2 class="h3 mt-4"><%= t('.add_sharing') %></h2>
```

Comment:
Reading notes: Those curious `H2` that wanna look like `H3`…kind of wonder how mangled the HTML actually is.

Reply:
Yeah these are curious indeed, my goal is to minimize the difference between our Hyrax overrides and Hyrax's implementation so a lot of it I just copied over.

https://github.com/samvera/hyrax/blob/double_combo/app/views/hyrax/base/_form_share.html.erb#L2